lib/appoptics_apm/api/util.rb

```diff
@@ -46,28 +46,6 @@ def trim_backtrace(backtrace)
         backtrace[0, 180] + ['...[snip]...'] + backtrace[backtrace.size - 20, 20]
       end
 
-      # Internal: Check if a host is blacklisted from tracing
```

Comment:
This was here for almost a decade: https://github.com/appoptics/appoptics-apm-ruby/commit/fdee534a7af1ab99d67ffa9c40a1d88522aad880 and now it is gone...